### PR TITLE
network: Deprecate bridged networking mode.

### PR DESCRIPTION
**Pull request by amshinde**
We plan to get rid of this feature in the future.

Fixes: #1950

Signed-off-by: Archana Shinde <archana.m.shinde@intel.com>

### CODE REVIEW

File: cli/config/configuration-fc.toml.in [hunk 1]
```diff
@@ -289,9 +289,11 @@ path = "@NETMONPATH@"
 # the container network interface
```
Comment (devimc) [inline cli/config/configuration-fc.toml.in]: could you please also update the acrn file (configuration-acrn.toml.in) ?

Comment (amshinde) [reply cli/config/configuration-fc.toml.in]: done

File: Makefile [hunk 2]
```diff
@@ -300,7 +300,7 @@ ifneq (,$(ACRNCMD))
 
     # acrn-specific options (all should be suffixed by "_ACRN")
     DEFBLOCKSTORAGEDRIVER_ACRN := virtio-blk
-    DEFNETWORKMODEL_ACRN := bridged
+    DEFNETWORKMODEL_ACRN := macvtap
```
Comment (amshinde) [inline Makefile]: cc @vijaydhanraj

**Comment by devimc** [comment]
cc @vijaydhanraj

**Pull request merged**